docs/developer-guide/internazionalization.md

```diff
@@ -0,0 +1,63 @@
+# Introduction
+MapStore2 offers the support for internationalization (I18N). To provide this functionality MapStore uses [react-intl](https://github.com/yahoo/react-intl).
+In this section you can find which configuration and JS files are involved in the I18N system.
+
+## How internationalization works in MapStore2
+MapStore2 first checks the browser's language. If it is not supported, MapStore2 will use its default language (english) or the first one available, as last choice.
```

Comment:
.... If the browser's language is not supported MapStore2 will be visible in english, if present, or the first language avalilable ...